### PR TITLE
Overlay for Unihertz Luna

### PR DESCRIPTION
**Pull request by PeterCxy**

### CODE REVIEW

File: overlay.mk [hunk 1]
```diff
@@ -285,6 +285,7 @@ PRODUCT_PACKAGES += \
 	treble-overlay-umidigi-power \
 	treble-overlay-unihertz-jelly2 \
 	treble-overlay-unihertz-titanpocketeea \
+	treble-overlay-unihertz-luna \
```
Comment (AndyCGYan) [inline overlay.mk:287]: `Fatal: overlay.mk: Keep entries sorted`

File: Unihertz/Luna/res/values/config.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item type="dimen" name="config_screenBrightnessSettingDefaultFloat">0.4</item>
+    <item type="dimen" name="config_screenBrightnessSettingMaximumFloat">1.0</item>
+    <item type="dimen" name="config_screenBrightnessSettingMinimumFloat">0.0</item>
+    <item type="dimen" name="config_signalCutoutHeightFraction">10.0</item>
```
Comment (AndyCGYan) [inline Unihertz/Luna/res/values/config.xml]:
```
Fatal: ./Unihertz/Luna/res/values/config.xml: defines a non-existing attribute config_screenBrightnessSettingDefaultFloat
Fatal: ./Unihertz/Luna/res/values/config.xml: defines a non-existing attribute config_signalCutoutHeightFraction
Fatal: ./Unihertz/Luna/res/values/config.xml: defines a non-existing attribute config_signalCutoutWidthFraction
Fatal: ./Unihertz/Luna/res/values/config.xml: defines a non-existing attribute quick_qs_total_height
Fatal: ./Unihertz/Luna/res/values/config.xml: defines a non-existing attribute starting_surface_icon_size
Fatal: ./Unihertz/Luna/res/values/config.xml: defines a non-existing attribute config_num_physical_slots
Fatal: ./Unihertz/Luna/res/values/config.xml: defines a non-existing attribute config_supportsMicToggle
```

**Comment by PeterCxy** [comment]
Some unneeded overlays removed. Pretty sure the rest are needed.

**Comment by phhusson** [comment]
LGTM though it requires updating knownKeys

**Pull request merged**